### PR TITLE
Moving into getting V0 working for POA

### PR DESCRIPTION
**Pull request by bastosmichael**
## Description of change
Turning Power of Attorney End Points for Claims V0 API

## Original issue(s)
https://vajira.max.gov/browse/API-1631

### CODE REVIEW

**Review by LindseySaari — APPROVED** [review]

File: modules/claims_api/app/controllers/claims_api/v0/forms/power_of_attorney_controller.rb [hunk 1]
```diff
@@ -36,8 +36,6 @@ def submit_form_2122
           end
 
           ClaimsApi::PoaUpdater.perform_async(power_of_attorney.id)
-          data = power_of_attorney.form_data
-          ClaimsApi::PoaFormBuilderJob.perform_async(power_of_attorney.id) if data['signatureImages'].present?
```
Comment (hinzed1127) [inline modules/claims_api/app/controllers/claims_api/v0/forms/power_of_attorney_controller.rb:40]: what's different about v0 from v1 that makes these removable here?

**Comment by hinzed1127** [comment]
Are there any changes needed to address this line of things to do original issue? 

> Adding MVI lookup using the headers we accept in order to fetch participant ID

File: modules/claims_api/app/controllers/claims_api/v0/forms/power_of_attorney_controller.rb (hunk 1 above)
Comment (bastosmichael) [reply modules/claims_api/app/controllers/claims_api/v0/forms/power_of_attorney_controller.rb:40]: I checked and most of the work was already done in the corresponding parent controller due to ITF so I was pleasantly surprised how little was necessary to implement this properly after tests.

**Comment by bastosmichael** [comment]
@hinzed1127 To answer your question, I checked and most of the work was already done in the corresponding parent controller due to ITF so I was pleasantly surprised how little was necessary to implement this properly after tests.

**Pull request merged**